.github/workflows/commit.yml

```diff
@@ -0,0 +1,35 @@
+---
+name: "Release the latest docker image"
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  push:
+    name: Build and push the Docker image
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Check out repository'
+        uses: actions/checkout@v2
+      - name: 'Login to Docker Hub'
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: 'Build and push the docker image'
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          tags: google/tsunami-security-scanner:latest
```

Comment:
Since this config pushes the image on every commit, maybe we could use a different tag other than "latest", like "dev" or "head"? We generally should point "latest" to the last "release" of the project.

Reply:
That's fine with me. In my projects I use latest as HEAD and stable as the latest release but I'll follow your lead.  I'll also add a versioned release when a tag is pushed

Reply:
Moved to `dev`